cms-2015-collision-datasets/code/create_hlt_trigger_path_records.py

```diff
@@ -116,18 +116,9 @@ def main():
         else:
             pass
 
-    # Filter away HeavyIon records for now. We have created them and reserved
```

Comment:
@katilp We may want to check and discuss live the results, because currently we produce:

![hlt-hi-path](https://github.com/cernopendata/data-curation/assets/517546/ef71c5b4-62a5-422c-ad06-54accedf8f10)


Note 13 TeV PbPb, and no "heavy-ion keyword"...